### DISCUSSION TITLE
Server Error

### DISCUSSION THREAD

**Discussion by superggfun** [vercel/next.js#67809] [Help]
### Summary

Hi,
When I try to set GraphQL url at WORDPRESS_API_URL=, it shows "Error: Invalid src prop (http://1.gravatar.com/avatar/1192be98f88cc00d59fd5468ea10dbc2?s=96&d=mm&r=g) on `next/image`, hostname "1.gravatar.com" is not configured under images in your `next.config.js`
See more info: https://nextjs.org/docs/messages/next-image-unconfigured-host". Is that something problems?
![image](https://github.com/user-attachments/assets/7c84b18f-e3ef-4a26-b90b-5eaa9f068017)

**Comment by icyJoseph** [comment]
That error is because next/image is receiving a src prop that points to that domain. You need to "allow" such a thing, it is a security feature. You do that by setting the domain in the next.config.js file. 

See more here, https://nextjs.org/docs/app/building-your-application/optimizing/images#remote-images

**Reply by superggfun** [reply]
> That error is because next/image is receiving a src prop that points to that domain. You need to "allow" such a thing, it is a security feature. You do that by setting the domain in the next.config.js file.
> 
> See more here, https://nextjs.org/docs/app/building-your-application/optimizing/images#remote-images

Thanks for your help. I already solve by GPT.
```
//net.config.js
if (!URL.canParse(process.env.WORDPRESS_API_URL)) {
  throw new Error(`
    Please provide a valid WordPress instance URL.
    Add to your environment variables WORDPRESS_API_URL.
  `);
}

const { protocol, hostname, port, pathname } = new URL(
  process.env.WORDPRESS_API_URL,
);

/** @type {import('next').NextConfig} */
module.exports = {
  images: {
    remotePatterns: [
      {
        protocol: protocol.slice(0, -1),
        hostname,
        port,
        pathname: `${pathname}/**`,
      },
      {
        protocol: 'http',  // or 'https'
        hostname: '1.gravatar.com',
        port: '',
        pathname: '**',
      },
    ],
  },
};

```
